pkg/fixtures/dackbox.go

```diff
@@ -0,0 +1,1162 @@
+package fixtures
+
+import (
+	"github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/nodes/converter"
+)
+
+///////////////////////////////
+// Dackbox synthetic dataset //
+///////////////////////////////
+
+/////////////////////
+// Image with CVEs //
+
+// Data relationships
```

Comment:
not related at all: appreciate the diagram, helps better in understanding things 😸